pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "awesome-object-store"
-version = "2.2.1"
+version = "2.2.2"
```

Comment:
I think this should be a minor change instead of a patch change, patch like 0.0.z is for bugfixes, but this change adds new functionality so it should be a minor change like 0.y.0